lib/proto_dynamodb_kv/proto_dynamodb_kv_client.c

```diff
@@ -194,6 +197,157 @@ proto_dynamodb_kv_request_put(struct wire_requestqueue * Q, const char * key,
 	return (-1);
 }
 
+/**
+ * proto_dynamodb_kv_request_icas(Q, key, buf, buflen, buf2, buflen2,
+ *     callback, cookie):
+ * Send a request to change the value ${buf} (of length ${buflen}) to the value
+ * ${buf2} (of length ${buflen2}), associated with the key ${key} via the
+ * request queue ${Q}.  Invoke
+ *     ${callback}(${cookie}, status)
+ * upon request completion, where ${status} is 0 on success, 1 on failure, and
+ * 2 if the precondition failed.
+ * The values must be of length at most 256 kiB.
```

Comment:
Are you sure that you want a line break before this sentence?  I can't think of any other such linebreak in our API comments.

Reply:
Not sure at all.  Please fix.